docs/players-and-controllers/index.md

```diff
@@ -3,7 +3,7 @@ layout: default
 title: Overview
 ---
 
-## Hardware based
+## Squeezebox Hardware (Discontinued)
```

Comment:
Further down that section there's mention of SqueezeAMP. This would need to move elsewhere if you changed the title.